BUILD.md

```diff
@@ -51,6 +51,7 @@ aleph-node>`.
 ## Manual
 These are build dependencies we use in our linux images for `aleph-node`:
 ```
+rust-nightly-2021-10-24
```

Comment:
Not related to this change, but are there any significant changes to the dependencies version in the current production build? Ie, what below list looks like for a `aleph-node` binary that's running in our mainnet atm.

Reply:
Current `production` build is based on github's ubuntu-latest, so we kind of don't know which versions we are using. This PR will force build procedure to use fixed versions - if we ever want to use newer version of some package, we need to manually update `versions.nix` file.